components/molecules/TableData.jsx

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 
-const TableData = ({ className, children, context }) => (
+const TableData = ({ className, children, context, ...otherProps }) => (
     <td
         className={cx(className, {
             [`gds-table__cell--${context}`]: context
-        })}>
+        })}
+        {...otherProps}>
         {children}
     </td>
```

Comment:
Spreading otherProps as users may want to provide additional valid props like "colSpan"